documentation/announcing-az-module.md

```diff
@@ -170,12 +169,12 @@ For scripts that import modules directly, or use ```#Requires``` statements to s
 ### Installing Az and AzureRM Side-by-Side
 
 Az and AzureRM cannot be imported side-by-side into the same PowerShell session.  If you do not want to migrate your scripts from AzureRM to Az right away, there are two main options:
-- Install Az in PowerShell Core, and leave AzureRM in Windows PowerShell
+- Install Az in PowerShell 7, and leave AzureRM in Windows PowerShell
 - Install Az and AzureRM side-by-side in Windows PowerShell and ensure scripts do not mix the modules
 
-#### Install Az in PowerShell Core
-You can follow the instructions in [Installing PowerShell Core on Windows](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-6
-) to install PowerShell Core, then use ```Install-Module Az``` in PowerShell Core to acquire the Az module.  Since Windows PowerShell and PowerShell Core can exist side-by-side and do not share module directories, this will effectively isolate the two modules.
+#### Install Az in PowerShell 7
+You can follow the instructions in [Installing PowerShell 7 on Windows](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-6
```

Comment:
```suggestion
You can follow the instructions in [Installing PowerShell 7 on Windows](hhttps://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows
```

Reply:
Although doc site will redirect to latest version, will remove `?view=powershell-6`, `en-us` is unnecessary as user may from non-English courtry.